### ISSUE TITLE
Players can have white nametages in-game

### ISSUE THREAD

**Issue by CoWinkKeyDinkInc** [OvercastNetwork/Issues#1785]
This player was also moving, I'm not sure if he was an observer in the screenshot below, I didn't see him flying or anything.
![2015-04-07_19 14 34](https://cloud.githubusercontent.com/assets/7586568/7037157/1a6be548-dd5b-11e4-89d7-e55bb30cace7.png)

![2015-04-07_19 15 00](https://cloud.githubusercontent.com/assets/7586568/7037166/35911064-dd5b-11e4-8a62-d3ac9a1b9ef0.png)

**Comment by samtheman43** [comment]
It was the same guy, so I'm guessing he was just somehow bypassing a filter somehow.

**Comment by Brottweiler** [comment]
Related/duplicate? https://github.com/OvercastNetwork/Issues/issues/1074

**Comment by CoWinkKeyDinkInc** [comment]
The user or911180 does not exist, it is probably a nicked player.

**Comment by ghost** [comment]
@Brottweiler This is duplicated of https://github.com/OvercastNetwork/Issues/issues/1074, he´s a nicked player

**Comment by CoWinkKeyDinkInc** [comment]
@TiTo-418 But this player was moving, meaning he was still in the game.

**Comment by ghost** [comment]
> The user or911180 does not exist, it is probably a nicked player.

**Comment by CoWinkKeyDinkInc** [comment]
The issue you're referring to mentions that the player logged off and it may be an issue with Minecraft.  But here it mentions that the user was moving.

**Comment by ghost** [comment]
Yes, the players who have a white name (tab and ingame) are nicked players and they can move, write in the chat, etc. as a normal player

**Comment by Electroid** [comment]
This doesn't happen anymore.

**Comment by Brottweiler** [comment]
![2016-01-04_19 15 14](https://cloud.githubusercontent.com/assets/1613137/12096763/f37493fc-b317-11e5-8d55-53cea64dede8.png)

When I was going to TP to him I noticed he had switched servers.

Rejoining the server "fixed" it.

**Comment by ghost** [comment]
I've had the same thing happen. TPing showed him as offline, RTPing showed him as having been on the last server I was on. Guessing this must be the cause of it?
